ray_on_golem/server/services/golem/golem.py

```diff
@@ -219,19 +216,15 @@ async def _create_stack(
                 self._golem,
                 demand_manager.get_initial_proposal,
                 plugins=(
-                    BlacklistProviderIdPlugin(PROVIDERS_BLACKLIST.get(payment_network, set())),
+                    ProviderBlacklistPlugin(payment_network),
                     *extra_proposal_plugins.values(),
                     ProposalScoringBuffer(
                         min_size=50,
                         max_size=1000,
                         fill_at_start=True,
                         proposal_scorers=(
                             *extra_proposal_scorers.values(),
-                            MapScore(
-                                partial(
-                                    self._score_with_provider_data, payment_network=payment_network
-                                )
-                            ),
+                            ReputationScorer(payment_network=payment_network),
```

Comment:
```suggestion
                            ReputationScorer(payment_network),
```
Let's keep a consistent argument providing across plugins.

Reply:
:woman_shrugging: